### DISCUSSION TITLE
python

### DISCUSSION THREAD

**Discussion by xXTorTank2611Xx** [googleapis/google-api-python-client#1997] [Q&A]
Bonjour je ne trouve pas le fichier python dans le nouveau fichier non compressé que vous avez mis à jour hier.
Si quelqu'un peut m'orienter c'est avec plaisir.

En vous souhaitant une bonne journée

**Comment by lown44** [comment]
Bonjour, bonsoir je débute également mais tu peut aller sur la barre de recherche et tu tape "python" et ça devrais marcher 
Bonne journée, bonnne soirée
Lown 13 ans

**Comment by arthuRHD** [comment]
Hello, le problème est toujours d'actualité ?
